project/Common.scala

```diff
@@ -11,7 +11,10 @@ import com.typesafe.sbt.SbtScalariform.ScalariformKeys
 
 object Common extends AutoPlugin {
 
-  val AkkaVersion = "2.4.12"
+  val AkkaVersion = sys.env.get("akka") match {
+    case Some("2.5") => "2.5.1"
+    case _ => "2.4.18"
+  }
```

Comment:
ok, so this is testing source compatibility, but not binary compatibility. better than nothing